homeassistant/components/solaredge_local/sensor.py

```diff
@@ -160,7 +160,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     # Changing inverter temperature unit.
     sensors = deepcopy(SENSOR_TYPES)
-    if status.inverters.primary.temperature.units.farenheit:
+    if status.inverters.primary.temperature.units.fahrenheit:
```

Comment:
How is this attribute spelled in the library?

Reply:
Reverted.